src/components/ScenarioParameters/ScenarioParameters.js

```diff
@@ -182,27 +196,23 @@ const ScenarioParameters = ({
                           indicatorColor="primary"
                           textColor="primary"
                           onChange={handleChange}
-                          aria-label="scenario parameters">
-                          <Tab label={t('commoncomponents.tab.scenario.parameters.upload.file', 'Upload File template')} value="upload_file_template" className={classes.tab}/>
-                          <Tab label={t('commoncomponents.tab.scenario.parameters.array.template', 'Array Template')} value="array_template" className={classes.tab}/>
-                          <Tab label={t('commoncomponents.tab.scenario.parameters.basic.types', 'Basic Types template')} value="basic_types" className={classes.tab}/>
+                          aria-label="scenario parameters"
+                      >
+                        {
+                          SCENARIO_PARAMETERS_TABS_CONFIG.map((tab) => (
+                            (SCENARIO_PARAMETERS_TABS_CONFIG[0].runTemplate.includes(currentScenario.data.runTemplateName)) &&
```

Comment:
Shouldn't SCENARIO_PARAMETERS_TABS_CONFIG[0] be replaced by tab ?
Otherwise, the runTemplates of the first item of the list will be used for all items